backend/tests/integration/paymentLogs.api.test.js

```
@@ -242,10 +271,8 @@
     });
 
     test('should handle missing/invalid JWT', async () => {
-      const res = await request(app)
-        .get('/v1/stripe/payment-logs/analytics');
+      const res = await request(app).get('/v1/stripe/payment-logs/analytics');
       expect([401, 500]).toContain(res.status);
```

ESLint

[error] 'expect' is not defined.

(no-undef)